app/routes/data-quality.ts

```diff
@@ -1,3 +1,191 @@
+        await this.store.query('agenda-item', {
+          filter: {
+            ':has:title': true,
+            ':has:description': true,
+            sessions: {
+              'governing-body': {
+                'administrative-unit': {
+                  location: {
+                    label: params.municipalityLabel || undefined,
+                  },
+                },
+              },
+            },
+          },
+          size: 1,
+        });
+
+      const totalCountAgendaItemsTreated: AdapterPopulatedRecordArrayWithMeta<AgendaItemModel> =
+        await this.store.query('agenda-item', {
+          filter: {
+            ':or:': {
+              sessions: {
+                ':has:started-at': true,
+                ':has:ended-at': true,
+              },
+            },
+            sessions: {
+              'governing-body': {
+                'administrative-unit': {
+                  location: {
+                    label: params.municipalityLabel || undefined,
+                  },
+                },
+              },
+            },
+          },
+          size: 1,
+        });
+
+      // get all classifications (governing bodies) from the store and dont get duplicates
+      const allGoverningBodyClassifications = await this.store.findAll(
+        'governing-body-classification-code'
+      );
+
+      // put all of the classifications through a loop to get the count of agenda items per classification
+
+      const agendaItemsPromises = allGoverningBodyClassifications
+        .filter(
+          // filter out duplicates
+          (classification, index, self) =>
+            self.findIndex((c) => c.label === classification.label) === index
+        )
+        .map(async (governingBody) => {
+          const agendaItems: AdapterPopulatedRecordArrayWithMeta<AgendaItemModel> =
+            await this.store.query('agenda-item', {
+              filter: {
+                sessions: {
+                  'governing-body': {
+                    'administrative-unit': {
+                      location: {
+                        label: params.municipalityLabel || undefined,
+                      },
+                    },
+                    classification: {
+                      label: governingBody.label,
+                    },
+                  },
+                },
+              },
+              size: 1,
+            });
+
+          return {
+            classification: governingBody.label,
+            count: getCount(agendaItems) || 0,
+          };
+        });
+
+      // resolve the promises
+      agendaItemsPerGoverningBodyClassification = await Promise.all(
+        agendaItemsPromises
+      );
+
+      agendaItemsPerGoverningBodyClassification =
+        agendaItemsPerGoverningBodyClassification.map((item) => ({
+          ...item,
+          percentage:
+            ((item.count || 0) / (getCount(totalCountAgendaItems) || 1)) * 50,
```

Comment:
**Nitpick:** `getCount(totalCountAgendaItems)` is executed 6 times. What about execute once and use the result everywhere?